pipeline/objects/function.py

```diff
@@ -39,12 +39,7 @@ def __init__(
         # TODO: Add verification that all inputs to function have a typing annotation,
         # except for "self"
         if "return" not in function.__annotations__:
-            raise Exception(
-                (
-                    "You must define an output type for a piepline function. "
-                    "e.g. def my_func(...) -> float:"
-                )
-            )
+            function.__annotations__["return"] = None
```

Comment:
Does this mean [the check](https://github.com/mystic-ai/pipeline/pull/191/files#diff-a568846c25c8a6e1e7c05e50a643a924ea050b060589aba23ec9d6061d0b60c7R35) is redundant?

Reply:
It does yeah, I added this in for some stuff I want to try in the next 2 releases.